qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -13,8 +13,9 @@
 import tempfile
 import traceback
 
+from . import util
+from . import coverage
 from authproxy import AuthServiceProxy, JSONRPCException
-from util import *
```

Comment:
I'd prefer (ii), but if you want to revert this and do it in another PR it's fine with me too. I don't like (iii) because it implies changing this now and changing this back again later.